### ISSUE TITLE
logits and weights

### ISSUE THREAD

**Issue by ShengyuH** [vcg-uvic/learned-correspondence-release#30]
hi,

Given output O, you use tanh(ReLU(O)) as weights/confidence and Sigmoid(O) as logits used in binary cross-entropy loss. This will cause inconsistency in the optimization, can you justify this choice?

**Comment by kmyid** [comment]
We tried both and tanh(relu)) worked better in this case. The rationale behind was that it allows the weights to become completely zero, while using sigmoid would unable to turn something off completely. However, in our recent paper ACNe, with a better network architecture, you don't need this trick.

**Comment by ShengyuH** [comment]
Thanks for your prompt reply!
